src/API/Google/AdsAssetGroup.php

```diff
@@ -283,6 +284,69 @@ protected function get_assets( array $asset_groups ): array {
 		return $asset_groups;
 	}
```

Comment:
💅  `use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\AdsAssetGroupAsset;`

Not being used